### ISSUE TITLE
[Flytectl Feature] flytectl status should show env vars to be exported

### ISSUE THREAD

**Issue by kumare3** [flyteorg/flyte#1523]
**Describe the feature/command for flytectl**
if you use flytectl sandbox, as a user you may lose the information about how to use kubectl & flytectl to connect to flyte-sandbox. We should allows users to recover this using
`flytectl sandbox status`

**Provide a possible output or ux example**
```
$ flytectl sandbox status
...
To Connect to FlyteSandbox Kube cluster export:
...
To connect to FlyteSandbox using flytectl export:
...
```

**Comment by evalsocket** [comment]
@kumare3 there is no use of it until we don't fix the restart

**Comment by kumare3** [comment]
> @kumare3 there is no use of it until we don't fix the restart

I do not agree. If I start the sandbox and forget to export the env vars to connect to k8s or Flyte, there is no way way to recover the information. It is possible, I restart my terminal or just clear the history

**Comment by evalsocket** [comment]
Sure Ketan, I got your point
